### PR TITLE
Updated Steering Committee 

### PR DESCRIPTION
**Pull request by branflake2267**
Added Justin Hickman and modified Colin's Biz

### CODE REVIEW

**Comment by branflake2267** [comment]
Hi Google Bot

**Comment by jDramaix** [comment]
:) It seems he is awake now :p

File: src/main/markdown/steering.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ The steering committee consists of the following members (no particular order):
 
 * Ray Cromwell, [Google](https://www.google.com/about/)
 * Artur Signell, [Vaadin](http://www.vaadin.com)
-* Colin Alworth, [Sencha](http://www.sencha.com)
+* Justin, Hickman, [Sencha](http://www.sencha.com)
+* Colin Alworth, [RedCurrent](http://www.redcurrent.com)
```
Comment (jDramaix) [inline src/main/markdown/steering.md]: I think that for the "individual" member, we don't mention the company. @dankurka

Comment (dankurka) [inline src/main/markdown/steering.md]: If the company is not really involved we don't, but final say should be up to @niloc132

Comment (niloc132) [inline src/main/markdown/steering.md]: :+1: - I talked to Brandon and gave him the go-ahead to add this, with RedCurrent's name. We don't have *any* product shipped yet, so at least for now, as far as you know, all we do is GWT ;).

Eventually you would know better - we don't have any non-"internal only tool" app bits that are using GWT.

**Comment by jDramaix** [comment]
Thanks for the first pull request ;)

**Comment by branflake2267** [comment]
Cool, this is so nice. Nice Job!

**Pull request merged**